flask/cli.py

```diff
@@ -585,6 +585,8 @@ def load_dotenv(path=None):
     """
 
     if dotenv is None:
+        if path or os.path.exists('.env') or os.path.exists('.flaskenv'):
+            click.secho(' * TIP: $ pip install python-dotenv', fg='yellow')
```

Comment:
I've been trying to write more verbose messages, something like: `Tip: There are .env files present. Use "pip install python-dotenv to use them.`